cmd/chihaya/config.go

```diff
@@ -10,14 +10,10 @@ import (
 	"github.com/chihaya/chihaya/frontend/http"
 	"github.com/chihaya/chihaya/frontend/udp"
 	"github.com/chihaya/chihaya/middleware"
-
-	// Imported to register as middleware drivers.
```

Comment:
Does this not pass through golint or goimports anymore? That'd be sad, I think it's quite useful here tbh

Reply:
That's correct. I think it's obvious enough from the context, but the other options are:

- put the comment on the end of one of the import lines
- open new `import()` sections

Reply:
I think we should add more import blocks to keep the comments.